aptos-move/framework/aptos-stdlib/sources/math64.move

```diff
@@ -20,6 +23,16 @@ module aptos_std::math64 {
         }
     }
 
+    /// Returns a * b / c going through u128 to prevent intermediate overflow
+    public inline fun mul_div(a: u64, b: u64, c: u64): u64 {
+        (((a as u128) * (b as u128) / (c as u128)) as u64)
```

Comment:
curious why the outer most parenthesis?

Reply:
I noticed that move rejects x as u64 and only accepts (x as u64)